planner/core/resolve_indices.go

```diff
@@ -184,6 +184,27 @@ func (p *PhysicalSort) ResolveIndices() {
 	}
 }
 
+// ResolveIndices implements Plan interface.
+func (p *PhysicalWindow) ResolveIndices() {
+	p.physicalSchemaProducer.ResolveIndices()
+	p.ChildCols = p.Schema().Columns[:len(p.Schema().Columns)-1]
```

Comment:
Sometimes it may not be `i`. For queries like `select _tidb_rowid, sum(a) over() from t`, the schema of window functions is `_tidb_rowid, a, window_col`,   but the schema of table reader is `a, _tidb_rowid` because we will move the `rowid` to the last place in physical optimization.

Reply:
Why not just resolve the column in the schema.